### ISSUE TITLE
Can't set up a server

### ISSUE THREAD

**Issue by Zickzii** [Sunekaer/stoneBlock#390]
## Bug Report
I've been troubleshooting why my server won't launch and I have no clue what I'm doing, apologies in advance if this isn't the place to go asking for help but whenever I try LaunchServer.bat I just get a bunch of errors that make no sense to me as I don't understand anything about Java or programming

## Steps to Reproduce (for bugs)

Just open LaunchServer.bat in version 1.0.22r1

## Logs

* Client/Server Log: https://pastebin.com/VYyPX3K5
* Crash Log: null

## Client Information

* Modpack Version:1.0.22r1
* Server/LAN/Single Player:Server
* Optifine Installed:No?
* Shaders Enabled:No?

**Comment by jimmydorry** [comment]
Same. This version of Forge is likely looking for the server jar with a different name, but I don't know what it would be.

**Comment by jimmydorry** [comment]
Last working version for me is `StoneBlock-1.0.20` with `forge-1.12.2-14.23.5.2768-universal`. No issues there.

The issues appears to be with `forge-1.12.2-14.23.5.2768.jar`

**Comment by prozacgod** [comment]
Out of curiosity I pulled the latest forge from their site, same md5sum as the one in the zip file.

I did manage to get this version to launch - I took a heavy handed approach to this, I installed the latest DW20, assuming it had "idiot proof" setup - ran the FTBInstall.sh file.  Launched it fine.

Then I removed the libraries directory from my StoneBlock directory and swapped in the one that came from DW20 pack, relaunched and it was successful.

**Comment by knoxhack** [comment]
umm

**Comment by marcotc** [comment]
The correct server file seems to be `minecraft_server.1.12.2.jar` for me.

**Comment by ZachA77ack7** [comment]
Has there been any fix for this. I am having the same problem and cant figure out what to do to fix it.

**Comment by nzminecraftDOTcom** [comment]
I kinda cheated a little on this one
Copied the libraries and forge universal and minecraft_server.1.12.2.jar from there and plonked it into the 22R1 release and it seems to work.

**Comment by knoxhack** [comment]
Try this one guys:

https://cdn.discordapp.com/attachments/157898273808580608/502293113902071809/StoneBlock-1.0.22r1-Server.zip

**Comment by Brittlejf** [comment]
@knoxhack I can confirm that those files worked better using the original server files for 1.0.22r1. 
https://minecraft.curseforge.com/projects/stoneblock/files/2628376#additional-files
A few adjustments were required as seen below. 
```
mv Install.bat Install.sh
sed -i 's/minecraft_server.jar/minecraft_server.1.12.2.jar/' Install.sh
bash Install.sh
bash ServerLaunch.sh
```

**Comment by knoxhack** [comment]
@Brittlejf can you explain more? And what OS? I tested in windows 10 and it worked

**Comment by Brittlejf** [comment]
Sure, I am running it on a Linux server, Centos 7 with Java 8 (openjdk):
```
$ rpm -qa | grep openjdk
java-1.8.0-openjdk-1.8.0.181-3.b13.el7_5.x86_64
java-1.8.0-openjdk-headless-1.8.0.181-3.b13.el7_5.x86_64
```
I downloaded the server files that you just provided, but that was missing the following directories required for my world: mods, config, resources, libraries & scripts. 

Since I had a previous world I wanted up and running on the new version I also required the mod folders as well as the proper server files. 

I downloaded the 1.0.22r1 server files from curseforge as well to get the modpack directories.
Copied the directories mentioned above to the same folder where I downloaded the server files you provided. 
Then I ran the commands from my previous comment to start the server. 
It prompted me to run the `/fml confirm` command for missing blocks as expected during a version upgrade. 

I hope that explains it better.

**Comment by Mixalhsss** [comment]
Thanks a lot guys it actually worked, have a nice day!

**Comment by knoxhack** [comment]
Awesome, will be implement in next update

**Comment by ZachA77ack7** [comment]
Okay so just for  clarification if I’m running windows 10 and I also have an old server that I would like to bring up to date what all do I need to do other than download the server files from the link that @knoxhack gave? 

Sorry I’m rubbish with this kind of stuff I’m just trying to play some Minecraft lol

**Comment by sigfodrone** [comment]
Thanks, it works. Confirmed Windows10.
@ZachA77ack7 The following steps worked for me:
1. Extract the original server files to a location you choose.  
2. Extract the new server files offered by knoxhack above (https://cdn.discordapp.com/attachments/157898273808580608/502293113902071809/StoneBlock-1.0.22r1-Server.zip) and overwrite files in the original location from step 1. 
3. run the install.bat to download minecraft_server.jar
4. rename 'minecraft_server.jar' to 'minecraft_server.1.12.2.jar' 
5.  run the 'LaunchServer.bat'

**Comment by ZachA77ack7** [comment]
by original im assuming you mean the newest version of the server files..... how to i keep my world and everything from my old server when moving to this new one?

**Comment by nzminecraftDOTcom** [comment]
The folder named "world" assuming you've left the server.properties level-name=world as it comes by default.
That should be all you have to copy over to play the exact same world. You may need to run the /fml confirm command when starting your game, or click a button to accept block changes due to updates etc.

**Comment by ZachA77ack7** [comment]
okay thanks guys huge help!

**Comment by knoxhack** [comment]
Implemented in StoneBlock-1.0.23 [(https://minecraft.curseforge.com/projects/stoneblock/files/2629767)](https://minecraft.curseforge.com/projects/stoneblock/files/2629767)
